src/main/java/edition/academy/seventh/serivce/IPromotionScrapping.java

```diff
@@ -0,0 +1,22 @@
+package edition.academy.seventh.serivce;
+
+import edition.academy.seventh.database.model.Book;
+
+import java.io.IOException;
+import java.util.List;
+
+/**
+ *
+ * Provide API to get books on sale from given bookstore by scrapping.
```

Comment:
Provides API to get books on sale from given bookstore. Dokumentujemy CO robi metoda, a nie JAK to robi